app/controllers/api/articles_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::ArticlesController < ApplicationController
+  before_action :authenticate_user!, only: %i[create]
```

Comment:
what does %i do?

Reply:
The journalist needs to be authenticated when he/she creates an article.